display/d.mon/stop.c

```diff
@@ -14,6 +14,10 @@ static int stop(const char *);
 
 int stop_mon(const char *name)
 {
+    if (strlen(name) >= GPATH_MAX) {
+        G_fatal_error(_("Monitor name <%s> is too long."), name);
+    }
+
```

Comment:
Here, it is not constructing any strings, so a check is not needed.

Reply:
Removed. Done